content/docs/guide/content.md

```diff
@@ -136,8 +136,19 @@ data/other.txt                      3.55 MiB  257.24 MiB/s [####################
 Snapshot 13190897 created: 17 files, 2 dirs, 0 symlinks, 0 errors, 1.71 GiB Original Size, 1.42 GiB Storage Size
 ```
 
-Use the `-d, --description` flag in order to describe your changes from snapshot
-to snapshot.
+The `store` command offers a variety of useful flags:
+- `-c, --compression` lets you choose the compression algorithm used before
```

Comment:
Should probably read: "used to store your data"

Reply:
Also, let's keep this consistent and remove all line-ending full-stops (dots) at the end of each item in this list.